packages/flutter_tools/test/commands.shard/hermetic/doctor_test.dart

```diff
@@ -459,60 +491,55 @@ void main() {
     testUsingContext('validate verbose output format', () async {
       expect(await FakeDoctor(logger).diagnose(), isFalse);
       expect(
-        logger.statusText,
-        equals(
-          '[✓] Passing Validator (with statusInfo)\n'
-          '    • A helpful message\n'
-          '    • A second, somewhat longer helpful message\n'
-          '\n'
-          '[✗] Missing Validator\n'
-          '    ✗ A useful error message\n'
-          '    • A message that is not an error\n'
-          '    ! A hint message\n'
-          '\n'
-          '[!] Not Available Validator\n'
-          '    ✗ A useful error message\n'
-          '    • A message that is not an error\n'
-          '    ! A hint message\n'
-          '\n'
-          '[!] Partial Validator with only a Hint\n'
-          '    ! There is a hint here\n'
-          '    • But there is no error\n'
-          '\n'
-          '[!] Partial Validator with Errors\n'
-          '    ✗ An error message indicating partial installation\n'
-          '    ! Maybe a hint will help the user\n'
-          '    • An extra message with some verbose details\n'
-          '\n'
-          '! Doctor found issues in 4 categories.\n',
-        ),
-      );
-    }, overrides: <Type, Generator>{AnsiTerminal: () => FakeTerminal()});
+          logger.statusText,
+          equals('[✓] Passing Validator (with statusInfo) [0ms]\n'
```

Comment:
This seems to fail on CI - in the most recent run the times were 1ms:
```
Expected: '[✓] First validator title (A status message) [0ms]\n'
            '    • A helpful message\n'
            '    • A different message\n'
            '\n'
            '• No issues found!\n'
            ''
  Actual: '[✓] First validator title (A status message) [1ms]\n'
            '    • A helpful message\n'
            '    • A different message\n'
            '\n'
            '• No issues found!\n'
            ''
   Which: is different.
          Expected: ... message) [0ms]\n     ...
            Actual: ... message) [1ms]\n     ...
                                  ^
           Differ at offset 46
```

Suggest we match with a regex to allow any value for ms?

Reply:
This is actually a bug in the tests. A fix is going to land in #158124